### ISSUE TITLE
Inconsistent Parameter descriptions preventing deployment.

### ISSUE THREAD

**Issue by ssccio** [aws/copilot-cli#4882]
When creating multiple types of environment addons from `copilot storage init`, the Env description parameter is not consistent, causing an error in deployment:

```
ken@ip-192-168-1-224 test % grep -A2 "Env:" copilot/environments/addons/*
copilot/environments/addons/test-cluster.yml:  Env:
copilot/environments/addons/test-cluster.yml-    Type: String
copilot/environments/addons/test-cluster.yml-    Description: The environment name your service, job, or workflow is being deployed to.
--
copilot/environments/addons/testbucket.yml:  Env:
copilot/environments/addons/testbucket.yml-    Type: String
copilot/environments/addons/testbucket.yml-    Description: The name of the environment being deployed.
--
copilot/environments/addons/testtable.yml:  Env:
copilot/environments/addons/testtable.yml-    Type: String
copilot/environments/addons/testtable.yml-    Description: The name of the environment being deployed.
ken@ip-192-168-1-224 test % copilot env deploy
Only found one environment, defaulting to: test
✘ upload artifacts for environment test: parse environment addons: parameter logical ID "Env" defined in "test-cluster.yml" at Ln 6, Col 5 is different than in "testbucket.yml" at Ln 6, Col 5
```

**Comment by Lou1415926** [comment]
Hello @ssccio. Thank you for reporting this issue and love the quick contribution ❤️ !

The same issue was also reported in #4804 and fixed in #4813. The fix hasn't been released yet, but it will be ASAP. I looked at the fix that you sent in #4883 - it seems like all the `Description`s in the "from" diff are `The name of the environment being deployed.`, which are consistent with each other already because of #4813. After the next release, this shouldn't be an issue anymore!

**Comment by KollaAdithya** [comment]
Fix is now released in v1.28 https://github.com/aws/copilot-cli/releases/tag/v1.28.0
